hoomd/hpmc/integrate.py

```diff
@@ -311,12 +316,13 @@ class Sphere(_HPMCIntegrator):
     Args:
         seed (int): Random number seed
         d (float): Maximum move displacement, Scalar to set for all types, or a
-            dict containing {type:size} to set by type.
+          dict containing {type:size} to set by type.
```

Comment:
Actually this is false, and I think my bad. When set on construction, it is merely the default for every type. It can then be updated on a per type basis through `integrator.d['A'] = 1` or `integrator.d = dict(A=1, B=0.5)`.